src/content/en/tools/lighthouse/_toc.yaml

```diff
@@ -39,10 +39,32 @@ toc:
         path: /web/tools/lighthouse/audits/manifest-short_name-is-not-truncated
       - title: Page Contains Some Content When Its Scripts Are Not Available
         path: /web/tools/lighthouse/audits/no-js
+      - title: Page Does Not Automatically Request Geolocation On Page Load
```

Comment:
The current title matches the title in the audit source. If you want to change the audit source, I'll change this, else I want these doc titles to always match the audit source for SEO